### ISSUE TITLE
iOS really bugging out

### ISSUE THREAD

**Issue by ramzimalhas** [libimobiledevice/libimobiledevice#1636]
All sorts of issues, even with my mac m1

**Comment by parasew** [comment]
congrats for this bug report

**Comment by nikias** [comment]
The year is still fresh, but this is certainly a good candidate for the best reported issue!

**Comment by mexmer** [comment]
not really, it was posted on 24th December, so more like christmas present.

**Comment by nikias** [comment]
> not really, it was posted on 24th December, so more like christmas present.

That makes this ticket even more hilarious 🤣
